multi-agent/lib/mas/elements/nodes/claude_agent/claude_agent_node.py

```
@@ -0,0 +1,504 @@
+                    f"ClaudeAgent {self.uid}: "
+                    f"Failed to install skill from {repo_url} "
+                    f"(path: {skill_path}): {e}"
+                )
+
+    def _install_skill(
+        self, repo_url: str, skill_path: str, skills_dir: str
+    ) -> None:
+        """Clone a repo to temp, copy only the skill subfolder to skills_dir."""
+        tmp_dir = tempfile.mkdtemp(prefix="claude_skill_clone_")
+        try:
+            subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)